packages/docs/versioned_docs/version-2.0.0-beta-3/welcome.md

```diff
@@ -2,24 +2,54 @@
+Key UniRep features include:
+- **Large Proof Storage**: UniRep facilitates the storage of extensive proofs, such as proof of Ethereum address control from an ECDSA signature, which users can later retrieve and prove multiple times.
+- **Web2 Authentication Bridging**: Users authenticate once with a trusted entity via OAuth, enabling the entity to attest and store information about their web2 identity in zk format.
+- **Versatile Data Storage**: UniRep accommodates a wide array of data essential to applications, including user reputation, actions taken, group memberships, and more.
+- **Seamless Interoperability**: Applications can easily interconnect by enabling users to create proofs using publicly available state for user verifications.
+- **No Forced Data Sharing**: With UniRep, users are never forced to receive unwanted data from applications, maintaining their freedom of choice.
 
-UniRep can be used to store data important to applications. This might include reputation the user has accrued, actions the user has taken, groups the user is a member of, or anything else. Applications can interoperate by having users make proofs using publicly available state roots.
+<center><img src="/img/application.png" alt="UniRep: Private user data with &#x26; provable reputation" width="100%" /></center>
 
-**Read the latest UniRep blog post on [Privacy & Scaling Explorations](https://mirror.xyz/privacy-scaling-explorations.eth/FCVVfy-TQ6R7_wavKj1lCr5dd1zqRvwjnDOYRM5NtsE)**
+##  🤠  Design & Deploy a customized attester
+
+Attesters are at the application layer. They are the platforms, businesses, and communities in the ZK social ecosystem. They act as world builders and community managers. Attesters have great flexibility in what to build. They decide how many user identities are used, how users are onboarded, and how users interact with each other. Most importantly, attesters decide why someone receives a positive reputation and why someone receives a negative reputation. In other words, attesters provide accountability.
+
+:::info
+UniRep's key offering is providing developers with a pre-built, audited system to create apps that securely handle private user data and manage reputation using ZK technology. UniRep protocol allows applications to securely attest to user data changes through the use of dynamic, short-lived identifiers known as **epoch keys**.
+
+:::
+
+### Few inspriations for you to get started
+
+- Non-custodial shopping, streaming
+- Business review / recommendations
+- B2B reputation
+- P2P marketplace
+- P2P gig services
+- Product review
+- Video rating
+...and more.
```

Comment:
I think we can list unirep apps that is built by our team members
e.g. zketh, unirep social, zkdao, my badge, trustlist,...
we can include the github links and describe it a little bit

or put them in one of the categories 
e.g. 
- Web2 Authentication Bridging ([my-badge](https://github.com/kittybest/my-badge))
- P2P marketplace ([trustlist](https://github.com/CJ-Rose/trustlist))